bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -195,30 +196,30 @@ public boolean equals(Object obj) {
   private SerializableScan serializableScan;
 
   /**
-   * Creates a {@link CloudBigtableScanConfiguration} using the specified information.
+   * Creates a {@link CloudBigtableScanConfiguration} using the specified configuration.
```

Comment:
We can't just mention scan, there are other parameters that are part of the configuration.
We either don't mention each and every one and let @param tag do that or mention all of them.